wordle-web/src/views/WordleEditor.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <v-container class="d-flex justify-center align-center">
+    <v-card style="width: 75%">
+      <div class="bg-primary text-h5 pa-3 text-center">Wordle Editor</div>
+      <v-text-field label="Wordle" type="text" v-model="value" @input="searchLive" maxlength="5">
+      </v-text-field>
+      <v-row>
+        <v-col class="d-flex flex-left">
+          <v-btn
```

Comment:
![image](https://github.com/IntelliTect-Samples/EWU-CSCD379-2023-Spring/assets/43832670/83a0d83d-093d-4c1a-a76b-94f71b3163d0)

I seem to keep getting this error when trying to add a word, I think it's because you are doing a check to see if the searched word list is empty when adding a word... however, it never appears to be empty, so I can never actually add a word? 

You should always let the user hit your API so you can handle auth and logic checks on the backend and avoid doing checks like this on the front end. This would be better handled on the backend.

Reply:
Thanks for commenting on this, I don't know how I missed that when changing my search function.  To be completely honest, I'm really upset that I somehow completely forgot how I was handling that button when I changed how my search function worked to prevent moving to non-existent pages, proceeded to not think it would be a problem to never empty my search results, and then proceeded to not test adding words after changing it.  You're definitely right that I should have just let the call hit the API considering the API already knows how to handle duplicate words and just sending the request would have prevented this issue entirely (along with the countless others that would occur if we were to continue expanding the site and I were to continue my streak of not thinking my changes through).